advanced/lighttpd.conf.debian

```diff
@@ -12,6 +12,7 @@
 server.modules = (
 	"mod_access",
 	"mod_accesslog",
+	"mod_auth",
```

Comment:
It would work, but then there wouldn't be an already written guide (Jacob's blog post) to setting up password protection that people can be pointed to.